snapcraft/plugins/ant.py

```diff
@@ -59,22 +102,88 @@ def schema(cls):
             "items": {"type": "string"},
             "default": [],
         }
+
+        schema["properties"]["ant-version"] = {"type": "string"}
+
+        schema["properties"]["ant-version-checksum"] = {"type": "string"}
+
+        schema["required"] = ["source"]
```

Comment:
This is already two lines below.

Reply:
yeah, in maven too :-/
want me to remove it there?

Reply:
Darn, I missed that. Might as well.